src/platform/user/profile/vap-svc/actions/transactions.js

```diff
@@ -312,6 +313,9 @@ export const validateAddress = (
       event: 'profile-edit-failure',
       'profile-action': 'address-suggestion-failure',
       'profile-section': analyticsSectionName,
+      'error-key': `${error.errors?.[0]?.code}_${
```

Comment:
I chose to not de structure `errors` because we are returning the entire `error` object in the `dispatch`

Reply:
Thanks! This works for me. I'm excited to see what shows up in GA!